mirrord-cli/src/config.rs

```diff
@@ -19,9 +19,9 @@ pub(super) enum Commands {
 
 #[derive(Args, Debug)]
 pub(super) struct ExecArgs {
-    /// Pod name to mirror.
+    /// Target name to mirror.
     #[clap(short, long, value_parser)]
-    pub pod_name: String,
```

Comment:
Lets keep the old args available with a deprecation warning

Reply:
This will also let us defer the changes in the IDE extensions for now - make sure to test that this change didn't break them.

Reply:
are you saying we keep the previous functionality as well?

Reply:
The definition here is, we don't want anything using the old parameters to break.

Reply:
using clap, i believe, not sure, we can only provide one primary argument? i think it can be either of --target or --pod-name, still looking into clap

Reply:
What makes it a primary argument?

Reply:
i mean --pod-name is required, if a user does not specify one then mirrord exec does not work

Reply:
So this point is resolved with ArgGroup right?

Reply:
yup resolved now